apps/condo/domains/common/components/containers/GlobalStyle.tsx

```diff
@@ -295,9 +295,21 @@ export default function GlobalStyle () {
               ${cardCSS}
               
               .ant-notification-notice.tasks {
+                padding: 17px;
```

Comment:
Please avoid to use global

Reply:
It's not a global, It's only for notification with `tasks` CSS-class.
Since there is an Ant notification component, not a component from `TaskProgress.tsx` module, here is an only place to declare it.